.github/workflows/test.yml

```diff
@@ -47,6 +48,13 @@ jobs:
       id: pr-labels
       uses: joerick/pr-labels-action@v1.0.6
 
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v1
+      with:
+        platforms: all
+      if: runner.os == 'Linux'
```

Comment:
The user can select which (additional linux) platforms they want to support with the above action.

In other CIs, they can simply execute `docker run --privileged --rm tonistiigi/binfmt --install all` (or any other platform selection), as per [`binfmt` docs](https://hub.docker.com/r/tonistiigi/binfmt)

Reply:
We aren't running a test that uses this yet, though. Should we add a build with ARM or PowerPC? Possibly trying to not add too much, since it's probably 5-10x slower?